packages/docs-reanimated/docs/core/makeMutable.mdx

```diff
@@ -0,0 +1,157 @@
+---
+sidebar_position: 8
+---
+
+# makeMutable
+
+:::caution
+The usage of `makeMutable` is discouraged in most cases. It's recommended to use the [`useSharedValue`](/docs/core/useSharedValue) hook instead unless you know what you're doing and you are aware of the consequences (see the [Remarks](#remarks) section).
+
+`makeMutable` is used internally and its behavior may change over time.
+:::
+
+`makeMutable` is a function internally used by the [`useSharedValue`](/docs/core/useSharedValue) hook to create a [shared value](/docs/fundamentals/glossary#shared-value).
+
+It makes it possible to create mutable values without the use of the hook, which can be useful in some cases (e.g. in the global scope, as an array of mutable values, etc.).
+
+The created object is, in fact, the same as the one returned by `useSharedValue` hook, so the further usage is the same.
+
+:::info
+We use `mutable value` name for an object created by `makeMutable` to distinguish it from the `shared value` created by `useSharedValue`. Technically, `shared value` is a `mutable value` with an automatic cleanup.
```

Comment:
Let's move it to remarks section

too many info/caution boxes makes it hard to read the documentation

Reply:
and it seems `the code tags` are overused here. use code tags only to emphasize code. You can use _italic_ to emphasize other stuff
<img width="980" alt="image" src="https://github.com/user-attachments/assets/e801492d-6d47-4d49-ad77-6e4cadbc1398">